templates/konvoy-ui.yaml

```diff
@@ -0,0 +1,37 @@
+---
+apiVersion: kubeaddons.mesosphere.io/v1alpha1
+kind: Addon
+metadata:
+  name: konvoy-ui
+  namespace: kubeaddons
+  labels:
+    kubeaddons.mesosphere.io/name: konvoy-ui
+  annotations:
+    appversion.kubeaddons.mesosphere.io/konvoy-ui: "1.47.2"
+    endpoint.kubeaddons.mesosphere.io/konvoy-ui: /ops/portal/
+spec:
+  kubernetes:
+    minSupportedVersion: v1.15.0
+  cloudProvider:
+    - name: aws
+      enabled: true
+    - name: docker
+      enabled: true
+    - name: none
+      enabled: true
+  chartReference:
+    chart: kommander
+    repo: https://mesosphere.github.io/charts/stable
+    version: 0.1.10
+    values: |
```

Comment:
@s12chung just double checking, previously with the `kommander` chart no Values needed to be set as the defaults were all good. It would appear that's no longer the case?

Reply:
this is for `konvoy-ui`, it's the same chart, but different settings. `kommander` kubeaddons-configs has no values still.